litestar/_signature/types.py

```diff
@@ -9,3 +9,7 @@ class ExtendedMsgSpecValidationError(ValidationError):
     def __init__(self, errors: list[dict[str, Any]]) -> None:
         self.errors = errors
         super().__init__(errors)
+
+
+class SerializationWrapper:
```

Comment:
This doesn't seem to get used anywhere? Did you accidentally drop some commits along the way? :upside_down_face:

Reply:
ha yes, lemme remove it.